lib/search/query_builder.rb

```diff
@@ -128,7 +128,10 @@ def more_like_this_query_hash
       end
 
       {
-        more_like_this: { docs: docs }
+        more_like_this: {
+          docs: docs,
+          min_doc_freq: 0, # This overrides the new ES 2.4 default with the ES 1.7 default so that we can complete the migration
```

Comment:
Do we have a card to revisit this?

Reply:
Not at this stage. It's an odd choice they have made with this as it means how the cluster is balanced can have an impact on `more_like_this` results for low volume terms.

Having the default as 0 feels like the right approach, I never investigated the reasons for the change, as such I might change the message text to explain what it does better and remove the reference to getting tests to pass.

Thoughts?